cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,133 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import checkout from '../../support/fragments/checkout/checkout';
+import searchPane from '../../support/fragments/circulation-log/searchPane';
+import circulationRules from '../../support/fragments/circulation/circulation-rules';
+import noticePolicy, { NOTICE_ACTIONS } from '../../support/fragments/circulation/notice-policy';
+import noticePolicyTemplate, { TEMPLATE_CATEGORIES } from '../../support/fragments/circulation/notice-policy-template';
+import patronGroups from '../../support/fragments/settings/users/patronGroups';
+import topMenu from '../../support/fragments/topMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import checkInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+// TODO email checking
+describe('Recieving notice: Checkout', () => {
```

Comment:
let's add jira ticket about it and move in into backlog. Provide link to the ticket into this comment

Reply:
https://issues.folio.org/browse/FAT-1854

Reply:
add link to this ticket in TODO